File: Napakalaki/lib/prize.rb
# encoding: utf-8

module NapakalakiGame

  # class Prize: representa el buen rollo de un monstruo
  class Prize
    attr_reader :treasures  # tesoros ganados
    attr_reader :levels     # niveles ganados

    def initialize(t, l)
      @treasures = t
      @levels = l
    end

    def to_s
      "    * Tesoros ganados: #{@treasures} \n    * Niveles ganados: #{@levels}"
    end

  end # Prize

end # NapakalakiGame

File: Napakalaki/src/NapakalakiGame/Prize.java
/**
 * Prize.java
 */

package NapakalakiGame;

/**
 * Representa el buen rollo de un monstruo.
 */
public class Prize {
    
    private int treasures;
    private int levels;
    
    public Prize(int t, int l) {
        treasures = t;
        levels = l;
    }
    
    public int getTreasures() {
        return this.treasures;
    }
    
    public int getLevels() {
        return this.levels;
    }

    @Override
    public String toString() {
        return "Treasures: " + Integer.toString(treasures) +
                "\nLevel: " + Integer.toString(levels);
    
    }
    
}
